src/types/api.ts

```diff
@@ -28,6 +28,13 @@ export interface Rate<K extends keyof RateKindMap> {
   Kind: K;
   Amount: string;
   Currency?: RateKindMap[K];
+  BoostedOffer?: BoostedRate;
+}
+
+export interface BoostedRate {
```

Comment:
why isn't this also `BoostedOffer`?

Reply:
thought it was more of a rate than an offer, but hmm I'm not sure, probably will change it to BoostedOffer, thanks!

Reply:
ok updated!